report_thesis/src/sections/proposed_approach/testing_validation.tex

```diff
@@ -2,34 +2,46 @@ \subsection{Validation and Testing Procedures for Model Evaluation}\label{subsec
 
 One of the primary challenges is preventing data leakage.
 As per concentration matrix $\mathbf{C}$ in Section~\ref{sec:problem_definition}, each target only has one ground truth concentration value per oxide.
 However, each target is shot at multiple locations, resulting in multiple instances of the same target in the dataset, as shown in Table~\ref{tab:final_dataset_example}.
-Although the intensity values vary for each location, they fundamentally represent different measurements of the same target.
+Although the intensity values vary for each location, they fundamentally represent measurements of the same target.
 If we were to randomly split the dataset, some locations from a target could end up in the testing set while others remain in the training set.
 This would cause data leakage, as the testing set would no longer consist solely of unseen targets.
 To prevent this, we ensure that each target is represented only once in the dataset by grouping data from all locations on a given target.
 
 Furthermore, the limited availability of data, as mentioned in Section~\ref{sec:problem_definition}, poses another significant challenge due to the difficult collection process.
-The dataset we use consists of 408 samples. This is a large dataset by \gls{libs} standards.
-However, despite this, there are only a few samples with extreme concentration values for the oxides in the targets.
-These extreme values present concentrations significantly different from the rest of the data points. 
+The dataset we use consists of 408 samples, which is relatively large by \gls{libs} standards.
+However, there are only a few samples with concentration values for the oxides in the targets that are either very high or very low compared to the rest of the data, which we refer to as \textit{extreme values}.
+These infrequent high and low concentration values can be problematic.
+If such values end up in the test set, the model may be evaluated on data points outside the range it was trained on.
+This situation can lead to an inaccurate assessment of the model's performance, as it might not handle these uncommon concentration ranges effectively.
 
 When performing a random split of the dataset into multiple folds for cross-validation, as well as for training and testing sets, this small number of extreme values can result in an uneven distribution.
 The presence or absence of these extreme values in any given fold can heavily influence the model's performance metrics.
-If extreme values are disproportionately allocated to either the training or testing sets, the resulting model may struggle to generalize accurately.
+If extreme values are disproportionately allocated to the testing set, the resulting model may struggle to generalize accurately.
 This uneven distribution can lead to models that perform well on the majority of the data but fail to predict accurately for these extreme concentration values, which are critical in many practical applications.
```

Comment:
```suggestion
This uneven distribution can lead to models that perform well on the majority of the data but fail to predict accurately for these extreme concentration values.
```

I think saying "which is critical in many applications" is a strong enough claim to warrant a source, but at the same time I dont think it adds anything to the sentence. Therefore, I suggest just removing it.

Reply:
I'm for this change, but then I'd like to hear your thoughts on this line:

> This is prudent because, in many practical applications, the reliability of the model on typical data points is more critical than its performance on rare extremes.

Context: https://github.com/chhoumann/thesis-chemcam/blob/a26dbc6d3d2a7fa5899e75a4ef988e10cae3a339/report_thesis/src/sections/proposed_approach/testing_validation.tex#L141

Reply:
Ah, forgot to add a comment about that cause I was so focused on the "Test Full" in the graphs.
Yes I think this falls in the same category.

Reply:
Just had a thought - Is it possible to maybe rephrase the second one more in terms of "We found that the reliability of the model.... etc" ? I think that could work here.

Reply:
Done